app/src/main/java/com/globant/practice/presentation/presenter/SplashPresenter.java

```diff
@@ -78,4 +59,14 @@ public void stopWaitTimeToChangeView() {
     private boolean isViewAttached() {
         return view != null;
     }
+
+    /**
+     * Detach the instance of the view and change the flag that indicates
+     * if is the first time that the application is invoked.
+     */
+    @Override
+    public void detachView() {
+        view = null;
```

Comment:
you must replace this statement to call `super` so BasePresenter execute it instead.